src/libs/ajax/leonardo/models/app-models.ts

```diff
@@ -0,0 +1,6 @@
+import { LeoRuntimeStatus } from 'src/libs/ajax/leonardo/models/runtime-models'
+
+
+export interface App {
+  status: LeoRuntimeStatus //TODO: fix this type
```

Comment:
There is a ticket tracking adding App types. I gave it an appropriate file, but its a bit out of scope to type this here

Reply:
Do we have the ticket number? Should we comment it here?

Reply:
Yeah... originally was pretty overzealous and the ticket stated to add types for apps/disks/runtimes all in one. I didn't do that as just Runtimes resulted in a lot of changes, so I'll make individual tickets and link here and delete the subtasks.